news/2021-11-04-skinners-bulletin-august-2021.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "Skinners' Bulletin: August 2021"
+date: 2021-11-04 11:00:00 +0000
+---
+
+Better late than never! The second issue of the Skinners' Bulletin is finally in your mailbox. 
+
+![](/wiki/shared/news/banners/skinners-bulletin.png)
+
+Welcome to the Skinners' Bulletin! If this is your first time reading one of these articles, we essentially pick through some helpful statistics and standout creations from the osu! skinning community to help bring some amazing creators into the spotlight. If you're a fan of customizing your osu! experience with only the best and brightest skins available, look no further!
```

Comment:
in my other review, i put everything to uk spelling for consistency because it was way too random before

```suggestion
Welcome to the Skinners' Bulletin! If this is your first time reading one of these articles, we essentially pick through some helpful statistics and standout creations from the osu! skinning community to help bring some amazing creators into the spotlight. If you're a fan of customising your osu! experience with only the best and brightest skins available, look no further!
```